stdlib/public/core/CTypes.swift

```diff
@@ -100,7 +98,19 @@ public typealias CLongDouble = Double
 public typealias CLongDouble = Double
 #endif
 #elseif os(OpenBSD)
+#if arch(x86_64)
```

Comment:
As you may or may not be aware, OpenBSD spells "x86_64" as "amd64". I'm not sure if this will actually be a problem on the platform. Let me pick this and I'll have an answer for you later tonight.

Reply:
Actually, this is quite likely fine as-is; I was able to check on cross-compile.

Reply:
This is the Swift spelling and should be mapped to `x86_64`.

Reply:
(Checked offline, this seems correct).